docs/source/jupyter/sdk.ipynb

```diff
@@ -183,8 +183,9 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "class MyMarketPlaceApp(MarketPlaceApp):\n",
-    "    def heartbeat(self) -> str:\n",
+    "from marketplace.app.v0 import MarketPlaceApp\r\n",
+    "class MyMarketPlaceApp(MarketPlaceApp):\r\n",
```

Comment:
Still not sure whether this is a good idea, but then we probably have to provide some kind of `app_type` parameter or so to the `get_app()` function.